precompiles/xvm/evm_sdk/XVM.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.8.0;
```

Comment:
Maybe rename `evm_sdk` to `contracts`?

Reply:
Don't want to change layout. It would harder to keep up a fork.